src/vellum/workflows/nodes/displayable/inline_prompt_node/node.py

```diff
@@ -30,9 +30,11 @@ class Outputs(BaseInlinePromptNode.Outputs):
         The outputs of the InlinePromptNode.
 
         text: str - The result of the Prompt Execution
+        json: Optional[Dict[Any, Any]] - The result of the Prompt Execution in JSON format
```

Comment:
[nit] I think we want to start using `undefined` instead of `None` to represent "this output is missing"